src/Hl7.Fhir.Base/ElementModel/ScopedNodeExtensions.cs

```diff
@@ -1,145 +0,0 @@
-﻿/* 
- * Copyright (c) 2017, Firely (info@fire.ly) and contributors
- * See the file CONTRIBUTORS for details.
- * 
- * This file is licensed under the BSD 3-Clause license
- * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
- */
-
-using Hl7.Fhir.Rest;
-using Hl7.Fhir.Support.Poco;
-using Hl7.FhirPath.Sprache;
-using System;
-using System.Linq;
-
-#nullable enable
-
-namespace Hl7.Fhir.ElementModel
```

Comment:
Do we consider this breaking generally? if they used the extension method syntax before, this would not be source breaking, only if they use ScopedNodeExtensions.Resolve(node, ...) this is breaking. 

[EDIT] In any case I moved it from elementModel to Model, so it is breaking nonetheless

Reply:
Most people won't do the explicit `ScopedNodeExtensions.Resolve(node, ...)`, but it is sometimes necessary, e.g. if these extension methods are in assemblies that are referenced with an `external alias` (which is a technique used to use R3/R4 etc versions of the SDK in a single application).  So, we don't have such a rule, but we might say: "If the extension method is in an assembly that is not a satellite (so in base or conformance), then such a namechange is ok".

But then again, why rename this in the first place?

Reply:
Moving it is something we *could* make bw compatible (dotnet has features for moving things to different assemblies), but we don't have to, since this is a major new release. Just make sure you put it in the wiki with breaking changes!